addon/components/fixtable-grid.js

```diff
@@ -222,7 +223,17 @@ export default Ember.Component.extend({
 
   visibleContent: Ember.computed('sortedFilteredContent', 'currentPage', 'pageSize', 'clientPaging',
     function fixtableGrid$visibleContent() {
-      let sortedFilteredContent = this.get('sortedFilteredContent') || [];
+      let sortedFilteredContent = this.get('sortedFilteredContent');
+
+      // ensure sortedFilteredContent is a supported type
+      let allowedTypes = [Array, Ember.ArrayProxy, DS.ManyArray];
+      let matchingTypes = allowedTypes.filter(function (type) {
```

Comment:
you could use .some if you wanted to be super clever.

Reply:
yeah!!